notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #45.            * ((MACS_DICT[adv_net_size] + MACS_DICT[victim_net_size]) / 2)
</pre><p>This is assuming half the nodes of A-MCTS are victim and half adversary? That should be ~approximately true but doesn't seem guaranteed. I'm fine with the assumption but might be worth flagging it explicitly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Added explicit comment detailing assumption here.